### PR TITLE
Implement upload es-next api (part of #240)

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @inikulin @AlexanderMoskovkin @VasilyStrelyaev

### CODE REVIEW

File: test/functional/fixtures/api/es-next/upload/test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+var expect = require('chai').expect;
+
+
+describe('[API] Upload', function () {
+    describe('t.uploadFile', function () {
+        it('Should upload the specified file', function () {
+            return runTests('./testcafe-fixtures/upload-test.js', 'Upload the file', { only: 'chrome' });
+        });
+
+        it('Should validate selector argument', function () {
```
Comment (VasilyStrelyaev) [inline test/functional/fixtures/api/es-next/upload/test.js]:
```
        it('Should validate the selector argument', function () {
```

File: test/functional/fixtures/api/es-next/upload/test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            return runTests('./testcafe-fixtures/upload-test.js', 'Invalid selector argument (uploadFile)', { shouldFail: true, only: 'chrome' })
+                .catch(function (errs) {
+                    expect(errs[0]).contains('The selector is expected to be a string, but it was undefined.');
+                    expect(errs[0]).contains('> 28 |    await t.uploadFile(void 0, \'../test-data/file1.txt\');');
+                });
+        });
+
+        it('Should validate filePath argument', function () {
```
Comment (VasilyStrelyaev) [inline test/functional/fixtures/api/es-next/upload/test.js]:
```
        it('Should validate the filePath argument', function () {
```

File: test/functional/fixtures/api/es-next/upload/test.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                });
+        });
+    });
+
+    describe('t.clearUpload', function () {
+        it('Should clear the upload', function () {
+            return runTests('./testcafe-fixtures/upload-test.js', 'Clear the upload', { only: 'chrome' });
+        });
+
+        it('Should validate selector argument', function () {
```
Comment (VasilyStrelyaev) [inline test/functional/fixtures/api/es-next/upload/test.js]:
```
        it('Should validate the selector argument', function () {
```

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
Just found a style issue (it relates to the previous PR but can be fixed here):
Missed `'Error'` postfix in the [upload error](https://github.com/DevExpress/testcafe/blob/fa54252c100b55f500e22119348d36615218f492/src/errors/test-run/type.js#L37): 
```js
actionCanNotFindFileToUploadError:       'actionCanNotFindFileToUpload'
```

**Comment by AlexanderMoskovkin** [comment]
the same [here](https://github.com/DevExpress/testcafe/blob/fa54252c100b55f500e22119348d36615218f492/src/errors/test-run/index.js#L273)

**Comment by inikulin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**